Cargo.toml

```diff
@@ -26,7 +26,7 @@ travis-ci = { repository = "https://github.com/witnet/witnet-rust", branch = "ma
 ansi_term = "0.12.1"
 bytecount = "0.6.0"
 ctrlc = "3.1.3"
-directories = "2.0.2"
+directories-next = "1.0.2"
```

Comment:
Actually, this line can be removed

```suggestion
```

Reply:
Yes, I was not sure why that dependency was included there since it wasn't used, but I did not want to break anything.

Reply:
Yeah, ideally we would have some automated check that would detect unused dependencies, but we haven't found any.